packages/components/README.md

```diff
@@ -4,20 +4,19 @@
 
 React components made for creating applications
 
-# Installation
+## Install
 
-```sh
-# with npm
-npm install @caddijp/components
+    npm install @caddijp/components
```

Comment:
yarn add は削除ですか？ 特に反対とかはないですが、それに至った理由を知りたく！

Reply:
yarn support node.js modules resolution algorithm.
there is nothing we can do without yarn :)

> This will lay out your node_modules folder using Yarn’s resolution algorithm that is compatible with the node.js module resolution algorithm.

refs: 

- https://classic.yarnpkg.com/en/docs/migrating-from-npm
- https://nodejs.org/api/modules.html#modules_all_together